llap-server/src/java/org/apache/hadoop/hive/llap/io/decode/GenericColumnVectorProducer.java

```diff
@@ -220,6 +220,9 @@ public static List<TypeDescription> setTypeBuilderFromSchema(
         case TIMESTAMP:
           type.setKind(OrcProto.Type.Kind.TIMESTAMP);
           break;
+        case TIMESTAMP_INSTANT:
+          type.setKind(Type.Kind.TIMESTAMP_INSTANT);
```

Comment:
is this OrcProto.Type.Kind? if so, use it with the same convention as in other branches of this switch case

Reply:
Fixed.